frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -129,12 +129,9 @@ object Interpreter {
       val contents = Dag.toDotGraph(state.build.dags)
       logger.info(contents)
     } else {
-      // TODO: Pretty print output of show projects, please.
       val configDirectory = state.build.origin.syntax
-      logger.info(s"Projects loaded from '$configDirectory':")
-      state.build.projects.map(_.name).sorted.foreach { projectName =>
-        logger.info(s" * $projectName")
-      }
+      logger.debug(s"Projects loaded from '$configDirectory':")
+      state.build.projects.map(_.name).sorted.foreach(logger.info)
```

Comment:
What's the order of the projects?

Reply:
Whatever the order of reading them is. It's non-deterministic, if you're asking for that.

Reply:
They're printed in alphabetical order if that's what you're asking?

Reply:
alphabetical order is good IMO, having a deterministic order is good IMO

Reply:
Ah no! Sorry, it's alphabetical by default.